src/me/components/GettingStarted.tsx

```diff
@@ -22,9 +22,16 @@ import {AppState} from 'src/types'
 // Selectors
 import {getOrg} from 'src/organizations/selectors'
 
+// Utils
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+
 type ReduxProps = ConnectedProps<typeof connector>
 type Props = RouteComponentProps & ReduxProps
 
+if (isFlagEnabled('rudderstackReporting')) {
+  require('src/cloud/utils/rudderstack')
+}
```

Comment:
this relates to the previous comment but basically we aren't guaranteed to ever hit this page